client/src/components/PredictionPanel.vue

```diff
@@ -47,18 +56,26 @@ export default defineComponent({
       ScorecardSummaryMessages,
     };
   },
+  // TODO: Handle error state where there is no lead prediction
+  // after the API has returned.
   computed: {
-    // Predicted estimate of lead.
-    percentLead(): number | null {
+    // Predicted lead likelihood for parcel's public lines.
+    publicLeadLikelihood(): string | null {
```

Comment:
Note that low prediction is the same as high prediction in the DB right now bc we don't yet have intervals